api/files.js

```diff
@@ -1,18 +1,24 @@
 /* jshint esversion: 6 */
 
-var promise = require('bluebird');
-var fs = promise.promisifyAll(require('fs'), {
+var _Promise = require('bluebird');
```

Comment:
Any reason why you are using `_Promise` over `Promise`? Other files use `Promise`.